math/mathcore/test/CMakeLists.txt

```diff
@@ -74,6 +74,12 @@ ROOT_STANDARD_LIBRARY_PACKAGE(TrackMathCoreUnitDict
                               LINKDEF stress/TrackMathCoreLinkDef.h
                               DEPENDENCIES Core MathCore RIO GenVector)
 
+if(MSVC)
+  add_custom_command(TARGET TrackMathCoreUnitDict POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libTrackMathCoreUnitDict.dll
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libTrackMathCoreUnitDict.dll)
+endif()
+
```

Comment:
Nope :-) This is what I am talking about.

Reply:
Oops! right, sorry, and well, if we don't copy it, we have to add `${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>`  in the `PATH` in order to find/load the dll

Reply:
That may not be a bad idea, if that's how Windows finds libraries.